Makefile

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PROJECT          := ingress-default-backend
+PROJECT          := gardener/ingress-default-backend
 VERSION          := $(shell cat VERSION)
-REGISTRY         := eu.gcr.io/gardener-project/gardener
+REGISTRY         := europe-docker.pkg.dev/gardener-project/public
```

Comment:
Please move `gardener` to the `REGISTRY` as it was before.

Reply:
done